frontend/javascripts/oxalis/api/api_latest.js

```diff
@@ -1051,8 +1077,32 @@ class DataApi {
    * @example
    * api.data.setConfiguration("segmentationOpacity", 20);
    */
-  setConfiguration(key: $Keys<DatasetConfiguration>, value) {
-    Store.dispatch(updateDatasetSettingAction(key, value));
+  setConfiguration(key: $Keys<DatasetConfiguration> | OutdatedDatasetConfigurationKeys, value) {
+    console.warn(
+      `The properties segmentationOpacity and isSegmentationDisabled are no longer directly part of the data configuration. To still set 
+      them use the segmentation "layers" configuration. This object has settings for the segmentation layer with the properties "alpha" 
+      and "isDisabled". Use "const layerSettings = api.data.getConfiguration('layers'); layerSettings[<segmentationLayerName>].alpha = 40;
+      api.data.setConfiguration('layers', layerSettings);" to set these properties.`,
```

Comment:
I think, you can make it a bit shorter by omitting the part which explains the code, as that should be self-explanatory:
```
The properties segmentationOpacity and isSegmentationDisabled are no longer directly part of the data configuration. Instead, they are part of the segmentation layer configuration and can be set as follows:
const layerSettings = api.data.getConfiguration('layers');
layerSettings[segmentationLayerName].alpha = 40;
layerSettings[segmentationLayerName].isDisabled = false;
api.data.setConfiguration('layers', layerSettings);"
```